bin/redmapper_batch.py

```diff
@@ -278,10 +280,45 @@ def load_batchconfig(filename):
         jf.write(parsl_script)
 
     elif (batchconfig[batchmode]['batch'] == 'slurm'):
-        raise NotImplementedError("Basic slurm submission not implemented yet.  Use parsl")
+
+        mem_per_node = int(batchconfig[batchmode]['mpn'])
+        pixels_per_node = int(np.floor(mem_per_node / memory))
```

Comment:
We want to use floor for `pixels_per_node` here because we would rather leave extra memory on the node than over allocate.  For example, if we have 10gb of memory per node and each pixel takes 3gb, floor(10/3) gives 3 pixels per node, ceil(10/3) gives 4 pixels per node.  We want the 3 pixels per node, because only 1gb of memory would be left for the 4th pixel.  I'd be happy to leave this as a comment about this, or extract it to a method that makes it more clear.

Similarly, with `total_tasks` we need to use floor but now because slurm job arrays start from 0.  Another example, 10 `total_pixels` and 3 `pixels_per_node`: ceil gives 4 `total_tasks`, and floor gives 3.  We want 4 total jobs in the job array, which for slurm is (0, 1, 2, 3).  Using 4 would overallocate. 

I will do some reading into the performance of shifter and parallel, but I do know that the NERSC documentation highlights both shifter and parallel for things that need to scale.  

I can also try to make the code more self-documenting here.